diff --git a/psplash-config.h b/psplash-config.h
index 82bb76d..ce918d0 100644
--- a/psplash-config.h
+++ b/psplash-config.h
@@ -23,10 +23,10 @@
 #define PSPLASH_STARTUP_MSG ""

 /* Bool indicating if the image is fullscreen, as opposed to split screen */
-#define PSPLASH_IMG_FULLSCREEN 0
+#define PSPLASH_IMG_FULLSCREEN 1

 /* Position of the image split from top edge, numerator of fraction */
-#define PSPLASH_IMG_SPLIT_NUMERATOR 5
+#define PSPLASH_IMG_SPLIT_NUMERATOR 7

 /* Position of the image split from top edge, denominator of fraction */
 #define PSPLASH_IMG_SPLIT_DENOMINATOR 6
diff --git a/psplash-colors.h b/psplash-colors.h
index d701089..f3bf403 100644
--- a/psplash-colors.h
+++ b/psplash-colors.h
@@ -20,7 +20,7 @@
 #define _HAVE_PSPLASH_COLORS_H

 /* This is the overall background color */
-#define PSPLASH_BACKGROUND_COLOR 0xec,0xec,0xe1
+#define PSPLASH_BACKGROUND_COLOR 0x20,0x20,0x20

 /* This is the color of any text output */
 #define PSPLASH_TEXT_COLOR 0x6d,0x6d,0x70
